corehq/apps/ota/tests/test_search_claim_endpoints.py

```diff
@@ -42,7 +42,10 @@ def setUp(self):
             case_id=self.case_id,
             case_type=CASE_TYPE,
             case_name=CASE_NAME,
+            external_id=CASE_NAME,
+            user_id=OWNER_ID,
             owner_id=OWNER_ID,
+            update={'opened_by': OWNER_ID},
```

Comment:
I put it in there when I was trying to deserialize using CommCareCaseSQLSerializer. iirc, it wasn't setting it, and it was a required field. Checking now, though, I can't see where it's a required field, so I might be mistaken.

(In the end I couldn't avoid validation failing because the serializer checked whether `case_id` was unique. So (A) it was hitting the DB to check uniqueness and hitting the DB is what we were trying to avoid by deserializing, and (B) of course search results are always going to exist so `case_id` will never be unique.)